### ISSUE TITLE
Error reported by `maven-findbugs-plugin`

### ISSUE THREAD

**Issue by ace-han** [jenkinsci/coordinator-plugin#44]
Try to have all the violation reported by `maven-findbugs-plugin`, like 

https://jenkins.ci.cloudbees.com/job/plugins/job/coordinator-plugin/92/console

Currently, I use below command line to get around `findbugs` error 

And this `skip-findbugs-with-tests` defines in parent's pom.xml

```sh
mvn release:prepare release:perform -Dusername=xxx -Dpassword=****** \
    -P skip-findbugs-with-tests,doclint-java8-disable
```

**Comment by ace-han** [comment]
We can cd to project root and then type below commands to see those reported violations

```sh
mvn findbugs:gui
```

**Comment by ace-han** [comment]
fixed by 5185af1
